products/libllbuild/BuildSystem-C-API.cpp

```diff
@@ -572,7 +572,7 @@ class CAPIBuildSystem {
     return getFrontend().initialize();
   }
 
-  bool build(const core::KeyType& key) {
+  bool build(llvm::StringRef key) {
```

Comment:
Shouldn't this stay as KeyType?

Reply:
This is a case where we were relying on the fact that keyType was a std::string.   We were getting auto conversion to it and then to a stringref.   The build system layer this calls doesn't actually want a KeyType. It takes the string and constructs a Target build key out of it.

As such, I think it would actually be better to rename this method to buildTarget to describe what it actually does.   That is a much larger change, though.